src/pages/cli/toolbox/sol-balance.md

```
@@ -0,0 +1,59 @@
+
+### Check specific wallet balance
+
+```bash
+mplx toolbox sol-balance 7xKXtg2CW87d97TXJSDpbD5jBkheTqA83TZRuJosgAsU
+```
+
+## Output
+
+The command will display the SOL balance in a formatted output:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)